topics/single-cell/tutorials/scrna-case-jupyter_basic-pipeline/tutorial.md

```diff
@@ -88,7 +88,7 @@ import pandas as pd
 You can import files from your Galaxy history directly using the following code. This will depend on what number in your history the final annotated object is. If your object is dataset #4 in your history, then you import it with the following:
```

Comment:
```suggestion
You can import files from your Galaxy history directly using the following code. This will depend on what number in your history the final annotated object is. If your object is dataset #1 in your history, then you import it with the following:
```